Limelight/UIComputerView.m

```diff
@@ -30,6 +30,15 @@ - (id) init {
     _hostButton = [UIButton buttonWithType:UIButtonTypeCustom];
     [_hostButton setContentEdgeInsets:UIEdgeInsetsMake(0, 4, 0, 4)];
     [_hostButton setBackgroundImage:[UIImage imageNamed:@"Computer"] forState:UIControlStateNormal];
+    
+#if TARGET_OS_TV
+    UIImageView *backgroundIcon = [[UIImageView alloc] initWithImage:[UIImage imageNamed:@"Computer"]];
+    backgroundIcon.userInteractionEnabled = YES;
+    backgroundIcon.adjustsImageWhenAncestorFocused = YES;
+    backgroundIcon.clipsToBounds = NO;
+    [_hostButton addSubview:backgroundIcon];
```

Comment:
You can still see the original size host icon if you look closely during the animation. Can we animate this without need a separate button and image view?

Reply:
this change has been reverted